pages/chain/[chainSlug].tsx

```diff
@@ -111,10 +111,10 @@ const ChainPage: ThirdwebNextPage = ({
 
   const sanitizedChainName = chain.name.replace("Mainnet", "").trim();
 
-  const title = `${sanitizedChainName} (${chain.nativeCurrency.symbol}) | RPC, Smart Contracts, Blockchain SDKs`;
-  const description = `Deploy smart contracts to ${sanitizedChainName} and build dApps with thirdweb's SDKs. Discover ${
-    chain.nativeCurrency.symbol
-  } RPCs, ${chain.faucets?.length ? "faucets," : "dApps"} & explorers.`;
+  const title = `${sanitizedChainName}: RPC and Chain Settings`;
+  const description = `Use the best ${sanitizedChainName} RPC and add to your wallet. Discover the chain ID, native token, explorers, and ${
+    chain.testnet ? "faucet options" : "more"
```

Comment:
we need to check if there's a faucet, we might have testnets that we don't have faucets URLS, in that case you say "and more" like in mainnets